### PR TITLE
Initial seed

### PR DESCRIPTION
**Pull request by justindchan**
Postgres seed implemented, working on Mongo implementation and write to CSV

### CODE REVIEW

**Comment by justindchan** [comment]
@rfagg805 tagging for visibility

File: README.md [hunk 1]
```diff
@@ -1,23 +1,27 @@
...
 ## Related Projects
 
-  - https://github.com/haab-solutions/reservation-module
-  - https://github.com/haab-solutions/photo-gallery-module
-  - https://github.com/haab-solutions/reviews-module
-  - https://github.com/haab-solutions/recommendations-module
+  - https://github.com/Dragon-Scaling/Reservation_service
+  - https://github.com/Dragon-Scaling/photo_gallery_service
+  - https://github.com/Dragon-Scaling/Review_service
+  - hhttps://github.com/Dragon-Scaling/Recommendations_Service
```
Comment (inyhsu) [inline README.md:11]: typo "hhttps://github.com/Dragon-Scaling/Recommendations_Service"

File: README.md [hunk 2]
```diff
@@ -37,3 +41,10 @@ npm install -g webpack
 npm install
 ```
 
+## CRUD APIs
+
+ - post('api/property/:propertyId/photos')
+ - get('api/property/:propertyId/photos')
```
Comment (inyhsu) [inline README.md:47]: How about getting single Photo

**Review by inyhsu — CHANGES_REQUESTED** [review]
There is a typo in ReadMe.md

**Comment by justindchan** [comment]
@rfagg805 correcting typos and adding api route for single photo. also refactoring schema and seed to use just one table now.

File: README.md (hunk 2 above)
Comment (justindchan) [reply README.md:47]: fixed typos

**Pull request closed without merging**